equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/config/CrmFormDetailsCriteria.java

```diff
@@ -0,0 +1,40 @@
+package uk.gov.justice.laa.crime.equinity.historicaldata.config;
+
+import jakarta.annotation.Nullable;
+import lombok.NoArgsConstructor;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.jpa.domain.Specification;
+import uk.gov.justice.laa.crime.equinity.historicaldata.model.CrmFileModelInterface;
+
+import java.util.Arrays;
+import java.util.List;
+
+@Configuration
+@NoArgsConstructor
+public class CrmFormDetailsCriteria {
+    public Specification<CrmFileModelInterface> getSpecification(CrmFormDetailsCriteriaDTO crmFormDetailsCriteriaDTO) {
```

Comment:
formatting - New line after line 13
Are you using any formatters? like Google Java code format?

Reply:
No we are not. But I am not a big fan of adding extra lines after annotations. I prefer to have it stick to the class they are notating too. it feels to me like typing something like:
```
class
 
   MyClass 

    { 
    
    [...]
    ```

Reply:
There is a "formatter" checker  running on deployment, 2 actually, codeQL and SonarCloud